api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -274,6 +274,32 @@ public OrderGroup getOrderGroupByUuid(String uuid) throws DAOException {
 	public OrderGroup getOrderGroupById(Integer orderGroupId) throws DAOException {
 		return (OrderGroup) sessionFactory.getCurrentSession().get(OrderGroup.class, orderGroupId);
 	}
+
+	/**
+	 * @see OrderDAO#getOrderGroupsByPatient(Patient)
+	 */
+	@Override
+	public List<OrderGroup> getOrderGroupsByPatient(Patient patient) throws APIException {
+		if (patient == null) {
+			throw new APIException("Patient Cannot Be Null");
+		}
+		Criteria criteria = sessionFactory.getCurrentSession().createCriteria(OrderGroup.class);
+		criteria.add(Restrictions.eq("patient", patient));
+		return criteria.list();
+	}
+
+	/**
+	 * @see OrderDAO#getOrderGroupsByEncounter(Encounter)
+	 */
+	@Override
+	public List<OrderGroup> getOrderGroupsByEncounter(Encounter encounter) throws APIException {
+		if (encounter == null) {
+			throw new APIException("Encounter Cannot Be Null");
```

Comment:
Is there any reason why you are capitalising the first letter of each word in the exception message?